external/src/main/java/com/redhat/cloud/policies/engine/handlers/TriggersHandler.java

```diff
@@ -315,14 +315,15 @@ public void createFullTrigger(RoutingContext routing) {
             @DocResponse(code = 400, message = "Bad Request/Invalid Parameters.", response = ResponseUtil.ApiError.class),
             @DocResponse(code = 500, message = "Internal server error.", response = ResponseUtil.ApiError.class)
     })
-    @Timed(absolute = true, tags = {"path=/triggers/trigger/{triggerId}", "method=PUT"})
+    @Timed
+            //(absolute = true, tags = {"path=/triggers/trigger/{triggerId}", "method=PUT"})
     public void updateFullTrigger(RoutingContext routing) {
         routing.vertx()
                 .executeBlocking(future -> {
                     String tenantId = ResponseUtil.checkTenant(routing);
 
                     // TODO Temp counter used to investigate the engine instability, remove ASAP.
-                    metricRegistry.counter("policies_engine_updateFullTrigger", new Tag("account", tenantId)).inc();
+                    meterRegistry.counter("policies_engine_updateFullTrigger", "account", tenantId).increment();
```

Comment:
General comment: will the dynamic tags be available in Prometheus? It didn't work in notifications.

Reply:
using timer now